### PR TITLE
drop six

### PR DESCRIPTION
**Pull request by xiaoge1001**
do it. https://github.com/tlsfuzzer/python-ecdsa/issues/293

### CODE REVIEW

**Comment by a-detiste** [comment]
@xiaoge1001 thanks for your patch, it has been included on Debian

https://wiki.debian.org/Python3-six-removal

**Comment by a-detiste** [comment]
Actualy, six is not needed for `b("literal")` `b"literal"`also works on Python2;
these lines could already be now merged.

**Comment by tomato42** [comment]
yes change to `b'literal'` is definitely something we can do (and you can see, happens in new code), but there are things like `int_types` that are still needed for py2 compatibility.

**Comment by a-detiste** [comment]
Yes yes... I'm almost done withj my pr "All those moments will be lost in time, like tears in rain."

**Comment by a-detiste** [comment]
Reducing the diff of 80%~90% would make life of downstreams easier.

File: src/ecdsa/numbertheory.py [hunk 1]
```diff
@@ -12,8 +12,7 @@
 from __future__ import division
 
 import sys
-from six import integer_types, PY2
```
Comment (cjwatson) [inline src/ecdsa/numbertheory.py:15]: This breaks the `if GMPY2 or GMPY` case: it tries to do `integer_types = tuple(integer_types + (type(mpz(1)),))` and gets a `NameError` as a result.  I suggest adding `integer_types = int,` somewhere up here and reverting the other removals of `integer_types` in this file, as I just did in https://salsa.debian.org/python-team/packages/python-ecdsa/-/commit/3a95d8623d28d73c8c7877e6c3ddd5ee43e33ca4.

**Pull request closed without merging**